oep-template.rst

```diff
@@ -4,42 +4,46 @@ OEP-XXX: OEP Template
 
 .. This is the template to use when you start a new OEP.
 
-+---------------+-------------------------------------------+
-| OEP           | :doc:`OEP-XXX </oeps/oep-0xxx>`           |
-+---------------+-------------------------------------------+
-| Title         | <OEP title>                               |
-+---------------+-------------------------------------------+
-| Last Modified | <date string, in YYYY-MM-DD format>       |
-+---------------+-------------------------------------------+
-| Author        | <list of authors' real names and          |
-|               | optionally, email addresses>              |
-+---------------+-------------------------------------------+
-| Arbiter       | <OEP czar's real name>                    |
-+---------------+-------------------------------------------+
-| Status        | <Draft | Accepted | Deferred |            |
-|               | Rejected | Withdrawn | Final |            |
-|               | Replaced>                                 |
-+---------------+-------------------------------------------+
-| Type          | <Architecture | Best Practice |           |
-|               | Process | Product Enhancement>            |
-+---------------+-------------------------------------------+
-| Created       | <date created on, in YYYY-MM-DD format>   |
-+---------------+-------------------------------------------+
-| `Resolution`  | <links to any discussions where the final |
-|               | status was decided>                       |
-+---------------+-------------------------------------------+
-| `Replaces`    | <OEP number>                              |
-+---------------+-------------------------------------------+
-| `Replaced-By` | <OEP number>                              |
-+---------------+-------------------------------------------+
-| `References`  | <links to any other relevant discussions  |
-|               | or relevant related materials>            |
-+---------------+-------------------------------------------+
++-----------------+--------------------------------------------------------+
+| OEP             | :doc:`OEP-XXXX-YYYY-ZZZZ </oeps/oep-XXXX-YYYY-ZZZZ>`   |
```

Comment:
In the other OEPS, you've used the format `OEP-XXXX </oeps/oep-XXXX-YYYY-ZZZZ>`. Which one did you mean? Is the goal to add info just to the filenames? Or to both the filenames and the visible identifier?

Reply:
Good catch.  I'll update the template.  I meant to add info to just the filenames since that is absent today.